lib/mzgl/gl/Graphics.cpp

```diff
@@ -428,7 +222,7 @@ float Graphics::getStrokeWeight() {
 void Graphics::setStrokeWeight(float f) {
 	if (f != this->strokeWeight) {
 		this->strokeWeight = f;
```

Comment:
I used to write `this->` everywhere too - but someone pointed out to me about how easy it is to make errors when refactoring code when using this pointers, so I stopped doing it. Now I find it a bit visually jarring (Obviously this wasn't introduced on this PR, just commented on it because it stood out)

Reply:
I hate this-> too - will fix here